backend/fregepoc/settings.py

```diff
@@ -153,3 +157,18 @@
 # MISC
 
 DOWNLOAD_PATH = os.environ.get("DJANGO_DOWNLOAD_PATH", "/usr/fregepoc-tmp/")
+
+# REST FRAMEWORK
+
+REST_FRAMEWORK = {
+    'DEFAULT_PAGINATION_CLASS': 'rest_framework.pagination.LimitOffsetPagination',
+    'PAGE_SIZE': 100,
+    'DEFAULT_PERMISSION_CLASSES': ['rest_framework.permissions.IsAuthenticated'],
```

Comment:
Ultimately the API will be accessible via [API keys](https://florimondmanca.github.io/djangorestframework-api-key/), making it the default permission class. Additionally, we currently don't have any authentication endpoints set up (`/admin` does not provide DRF compatible authentication). It might be beneficial to get rid of this line.

Reply:
`/admin` *does* provide DRF-compatible authentication (it's even actually the default configuration): https://www.django-rest-framework.org/api-guide/authentication/#sessionauthentication

Reply:
What I meant by

> Additionally, we currently don't have any authentication endpoints set up (/admin does not provide DRF compatible authentication).

was that DRF does not provide default session authentication **endpoints**. The only way of session authentication is via the vanilla django **template** (which `admin` uses). Hovewer, while it is correct that DRF provides endpoints for `TokenAuthentication`, it is not listed in this commit.

As discussed with @apardyl - token authentication is from now on ommited indefinately, which makes API keys the de facto only mean of authentication to REST endpoints - although adding `SessionAuthentication` to the `DEFAULT_AUTHENTICATION_CLASSES` will indeed benefit debugging, and thus I think it should be done.

Reply:
Well... technically DRF does provide a [session auth endpoint](https://www.django-rest-framework.org/tutorial/4-authentication-and-permissions/#adding-login-to-the-browsable-api), but we don't include it in urlpatterns. Also, with current config you can use HTTP "Basic" authentication to access REST endpoints, so api keys are nowhere the only way.